app/blinken.js

```diff
@@ -45,9 +45,11 @@
   Blinken.prototype.getStatus = function(group_id, environment_name, environment_url) {
     var self = this;
     $.getJSON(environment_url + "/cgi-bin/icinga/status.cgi?servicestatustypes=20&jsonoutput=1", function(data) {
-      var service_status = data.status.service_status;
-      var critical_entries = service_status.filter(function(o) { return o.status === "CRITICAL" && o.has_been_acknowledged === false }).length;
-      var warning_entries = service_status.filter(function(o) { return o.status === "WARNING" && o.has_been_acknowledged === false }).length;
+      var service_status = data.status.service_status.filter(function(o) {
```

Comment:
non blocking nit: would be good to name this variable something more descriptive based on how it's filtered.

Reply:
Idiomatic JavaScript tried not to use single variable naming when iterating over arrays.

Here we could use `status` rather than `o`

Looks like the data structure that icinga brings back is kinda naff though

Reply:
Your nits have been picked